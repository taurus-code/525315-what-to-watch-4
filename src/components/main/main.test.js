import React from "react";
import renderer from "react-test-renderer";
import Main from "./main.jsx";

const dataPromo = {
  title: `The Grand Budapest Hotel`,
  genre: `Drama`,
  releaseDate: `2014`
};

const titleMovies = [`Fantastic Beasts`, `Bohemian Rhapsody`, `Macbeth`, `Fantastic Beasts`, `Bohemian Rhapsody`, `Macbeth`, `Fantastic Beasts`, `Bohemian Rhapsody`, `Macbeth`];


describe(`Renders Main components`, () => {
  it(`Render component Main`, () => {
    const tree = renderer.create(
        <Main
          promo={dataPromo}
          titleMovies={titleMovies}
          titleMovieClickHandler={() => {}}
        />
    ).toJSON;

    expect(tree).toMatchSnapshot();
  });
});
