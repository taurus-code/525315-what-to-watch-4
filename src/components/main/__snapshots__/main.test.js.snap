// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Main components Render component Main 1`] = `[Function]`;
